x-pack/plugins/cases/public/components/tag_list/index.tsx

```diff
@@ -76,7 +76,7 @@ export const TagList = React.memo(
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [onSubmit, submit]);
 
-    const { tags: tagOptions } = useGetTags();
+    const { data: tagOptions = [] } = useGetTags();
```

Comment:
At the moment the component is being used only on the case view page. Do we want to refetch the tags if we invalidate a query with the key `CASE_VIEW_CACHE_KEY`? If yes, may we pass the `CASE_VIEW_CACHE_KEY` to the hook?

Reply:
It is used in list and create. I added the invalidation of this cache key in the refresh case view page.